Angular.md

```diff
@@ -0,0 +1,37 @@
+# Angular CLI
+
+| Command                                     |  Meaning                                                                              |
```

Comment:
1. rename the file to `angular.md` all lowercase 
2. Please add cheat sheet metadata (refer other cheat sheets)